jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/OpenShellInContainerDialog.kt

```diff
@@ -53,18 +61,23 @@ class OpenShellInContainerDialog(
 
     override fun createCenterPanel(): JComponent = panel {
         row(message("ecs.execute_command_task.label")) {
-            resourceSelector(
-                EcsResources.listTasks(container.service.clusterArn(), container.service.serviceArn()),
-                connectionSettings,
-                ::task
-            ).constraints(growX, pushX)
-                .growPolicy(GrowPolicy.MEDIUM_TEXT)
-                .withErrorOnApplyIf(message("ecs.execute_command_task_comboBox_empty")) { it.selected().isNullOrEmpty() }
+            cell(taskList)
+                .bindItem(::task)
+                .errorOnApply(message("ecs.execute_command_task_comboBox_empty")) {
+                    it.selected().isNullOrEmpty()
+                }.columns(30)
```

Comment:
any reason we hardcode the column size?

Reply:
The width of the dialog becomes too long if we dont, since it takes the size of the entire task role arn text